trade-finance-manager-ui/server/types/view-models/problem-with-service-view-model.ts

```diff
@@ -0,0 +1,4 @@
+export type ProblemWithServiceViewModel = {
```

Comment:
Should we add the user to this and pass in to the template so they still have the username and sign out options in the top bar?

Reply:
whoops, added